1228-public_page/1530-files_with_administrative_agreement_is_not_visible.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Files with Administrative Agreement is not visible
```

Comment:
Have we followed the process defined in https://cucumber.io/docs/bdd/ to generate this Feature?

Reply:
what specifically are you questioning?

Reply:
E.g. is the formulation the outcome of a discovery workshop? https://cucumber.io/docs/bdd/discovery-workshop/

Reply:
Weeellll... A written scenario was given to me, and then we had a discussion on github about the changes that were made 😅